src/test/java/com/atlassian/db/replica/internal/StaticExperiment.java

```diff
@@ -0,0 +1,24 @@
+package com.atlassian.db.replica.internal;
+
+import com.atlassian.db.replica.spi.Experiment;
+
+public final class StaticExperiment implements Experiment {
```

Comment:
I guess `DynamicExperiment` or `FFDriverExperiment` would be implemented in the product?